core-services/prow/02_config/generate-boskos.py

```
@@ -881,13 +912,18 @@ def generate_config():
-    ignore_list = []
-    cps = {}
+    ignored_cps = CLUSTER_PROFILE_SETS_IGNORE['profiles']
+    cps = {
+        'cluster_profile_sets': {},
+        'tests_whitelist': {},
+    }
+
     for cps_name, cps_data in CLUSTER_PROFILE_SETS_CONFIG.items():
-        if not cps_name in ignore_list:
-            cps[cps_name] = list(cps_data.keys())
+        if not cps_name in ignored_cps:
```

Ruff (0.15.12)

[error] Test for membership should be `not in`

Convert to `not in`

(E713)